bb-aem-65/src/main/java/com/cognifide/qa/bb/aem/core/component/dialog/dialogfields/ContentFragmentPathBrowserImpl.java

```diff
@@ -0,0 +1,84 @@
+/*-
+ * #%L
+ * Bobcat
+ * %%
+ * Copyright (C) 2016 Cognifide Ltd.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.cognifide.qa.bb.aem.core.component.dialog.dialogfields;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.*;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import com.cognifide.qa.bb.qualifier.Global;
+import com.cognifide.qa.bb.qualifier.PageObject;
+import com.cognifide.qa.bb.wait.BobcatWait;
+import com.cognifide.qa.bb.wait.TimingsBuilder;
+import com.google.inject.Inject;
+
+/**
+ * Default implementation of {@link PathBrowser}
+ */
+@PageObject(css = Locators.AUTOCOMPLETE_CSS)
+public class ContentFragmentPathBrowserImpl implements ContentFragmentPathBrowser {
```

Comment:
The `DefaultXYZ` naming is more aligned with the rest fields

Reply:
Fixed